notebooks/textbook/CHSH_Inequality.ipynb

```diff
@@ -0,0 +1,462 @@
+      "text/plain": [
+       "<Figure size 800x500 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "%matplotlib inline\n",
+    "\n",
+    "fig, ax = plt.subplots(figsize=(8, 5))\n",
+    "plt.plot(angles, bell_values, \".\", label=\"Bell Inequality Value\")\n",
+    "\n",
+    "plt.grid(which=\"major\", axis=\"both\")\n",
+    "plt.yticks([-1.5, -1, 0, 1, 1.5], [\"$-1.5$\", \"$-1$\", \"$0$\", \"$1$\", \"$1.5$\"])\n",
+    "plt.xticks([i*np.pi/3 for i in range(7)], [\"$0$\", \"$\\pi/3$\", \"$2\\pi/3$\", \"$\\pi$\", \"$4\\pi/3$\",\"$5\\pi/3$\",\"$2\\pi$\"])\n",
+    "plt.xlabel(\"Angle\")\n",
+    "plt.ylabel(\"Reduction to Bell Inequality Value\");"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Run on a QPU\n",
+    "\n",
+    "To run CHSH inequality on a QPU, we replace the LocalSimulator with an AwsDevice. \n",
+    "To reduce the cost, we run the the experiment only the defualt angles which gives the maximum CHSH inequality value.\n",
+    "\n",
+    "The cost to run this experiment is \\$0.3 per task and \\$0.00035 per shot on the Oxford Quantum Circuits Lucy device. Since we have four circuits of 1000 shots each, the total cost is \\$2.60 USD."
```

Comment:
```suggestion
    "The cost to run this experiment is \\$0.3 per task and \\\\$0.00035 per shot on the Oxford Quantum Circuits Lucy device. Since we have four circuits of 1000 shots each, the total cost is \\\\$2.60 USD."
```